arlo-client/src/components/Audit/CalculateRiskMeasurement.tsx

```diff
@@ -58,7 +58,7 @@ const QRs: React.FC<{ electionId: string; boardIds: string[] }> = ({
       {boardIds.map(id => (
         <span key={id} id={`qr-${id}`}>
           <QRCode
-            value={`${window.location.origin}/election/${electionId}/board/${id}`}
+            value={`${window.location.origin}/election/${electionId}/audit-board/${id}`}
```

Comment:
Hmmm, maybe this slipped in in a prior change, but isn't this QR code supposed to link to `/auditboard/{passphrase}` ?

Reply:
dunno, they end up at the same place though. should i change it?

Reply:
so the URL you have should *not* authenticate the audit board user, I think? I wouldn't want someone seeing a URL on a screen be enough to log in as the audit board. The passphrase URL is the one that should set the cookie.